ddtrace/api.py

```diff
@@ -25,7 +25,13 @@ def __init__(self, hostname, port, headers=None, encoder=None):
 
         # overwrite the Content-type with the one chosen in the Encoder
         self._headers = headers or {}
-        self._headers.update({'Content-Type': self._encoder.content_type})
+        self._headers.update({
+            'Content-Type': self._encoder.content_type,
+            'Datadog-Meta-Lang': 'python',
+            'Datadog-Meta-Lang-Version': PYTHON_VERSION,
+            'Datadog-Meta-Lang-Interpreter': PYTHON_INTERPRETER,
+            'Datadog-Meta-Tracer-Version': '0.8.5',
```

Comment:
We can't do that otherwise we should always remember to update that value too before a release. It should get the value from the library itself so the `__version__`. To do that you must be sure that there is not a circular dependency otherwise the `__version__` will not be available at import time.